backend/de.metas.manufacturing/src/main/sql/postgresql/ddl/functions/PP_Product_BOM_Recursive_Report.sql

```diff
@@ -1,25 +1,21 @@
-drop function if exists PP_Product_BOM_Recursive_Report(numeric);
-create or replace function PP_Product_BOM_Recursive_Report(p_PP_Product_BOM_ID numeric)
-returns table
-(
-	Line text,
-	ProductValue varchar,
-	ProductName varchar,
-	QtyBOM numeric,
-	Percentage numeric,
-	UOMSymbol varchar
-)
-as
-$BODY$
-	select
-		t.Line,
-		t.ProductValue,
-		t.ProductName,
-		t.QtyBOM,
-		t.Percentage,
-		t.UOMSymbol
-	from PP_Product_BOM_Recursive(PP_Product_BOM_Recursive_Report.p_PP_Product_BOM_ID, null) t
-	order by t.path
-$BODY$
-LANGUAGE sql STABLE
-COST 100;
+CREATE OR REPLACE FUNCTION pp_product_bom_recursive_report(p_pp_product_bom_id numeric)
+    RETURNS TABLE(line text, productvalue character varying, productname character varying, qtybom numeric, percentage numeric, uomsymbol character varying, supplier text)
```

Comment:
@mi-ps  pls fix the formatting. 
(maybe pressing ctrl+alt+L in intelliJ would help)